### ISSUE TITLE
[Feature Request] Vuetify 3 Vertical Stepper

### ISSUE THREAD

**Issue by joshvanallen** [vuetifyjs/vuetify#18842]
### Problem to solve
In Vuetify 2, there's an implementation of a Vertical Stepper and hasn't been implemented for Vuetify 3. This results in apps being redesign, finding ways to break the Stepper and tons of custom CSS, or just staying with Vuetify & Vue 2.
Looking at the [Material design spec](https://m1.material.io/components/steppers.html#steppers-usage), it seems to be a valid pattern to have a vertical option for stepper. 

### Proposed solution
Follow the Angular implementation of Stepper where there's an `orientation` prop and the accepted values are `horizontal` and `vertical`. The prop would default to `horizontal`. 
From the slot layout perspective, I'm not too sure if the existing implementation could work.

**Comment by MarcHartwig13** [comment]
According to [this comment](https://github.com/vuetifyjs/vuetify/pull/17685#issuecomment-1660772873) on the original issue to port VStepper to Vuetify 3 there are plans to make a specific vertical stepper component. I don't believe there has been any communication on when that component might be worked on.

I'm also waiting on this component so hopefully they'll begin work soon.

**Comment by joshvanallen** [comment]
@MarcHartwig13 ah thanks for that insight! I was just looking for things based on issue search and couldn't find anything about it. Should of looked in the original Stepper issue, oh well!

IMO, think this issue should still open (not directing at you Marc but if anyone else stumbles upon this) as more of a backlog item.

**Comment by gigerIT** [comment]
We're able to implement vertical steppers in v3 like this:
https://play.vuetifyjs.com/#eNq9VMtOwzAQ/JWVL4BKEvo4oKqpQNy5wI1wcJytatVJLMdNkar+O+u4FBfaQ6Gqcoh3PZ4Zj6x9W7PGiORR67hdIhuzicVSK25xmlUAkzbiWnfLrhB1Zbms0GxbXbOxqDUaaKOyLlClGXOdjO0wISqaIy+C8z92JcmDlVYh0by6f8Zg3HK1dI0+FVhIy3OFkIT8yQGBg/IrWRX1CgppUFhZV8TaorFScEXkhJvXq+0N0pT0jhn1PN5v6G8KT5QRVnbPU4A+4tojOtf+O0N0MAjDc8UFMiOZkzJz+F1mg3+Fdo7IhmFkrrhAZCRzUmQOv4ts+MfIfm0FVyEDua1AKN40pFfayD2eB6GkWGxdRxGZeKEF5FwsHA2dOI2i1yOKZ/yw4Kpvii9f+8PGdfwsmiTBjKKyEUZqIkG79KNKlro2FtZgcAYbmJm6hCsab1feHvE2XhNSB7nu3zhST0MEbHPLhvEovotytDweMV/es/dPvGamSg==

**Comment by johnleider** [comment]
duplicate of #5338

**Comment by garypyne** [comment]
> We were able to implement vertical steppers in v3 like this: https://play.vuetifyjs.com/#eNq9VMtOwzAQ/JWVL4BKEvo4oKqpQNy5wI1wcJytatVJLMdNkar+O+u4FBfaQ6Gqcoh3PZ4Zj6x9W7PGiORR67hdIhuzicVSK25xmlUAkzbiWnfLrhB1Zbms0GxbXbOxqDUaaKOyLlClGXOdjO0wISqaIy+C8z92JcmDlVYh0by6f8Zg3HK1dI0+FVhIy3OFkIT8yQGBg/IrWRX1CgppUFhZV8TaorFScEXkhJvXq+0N0pT0jhn1PN5v6G8KT5QRVnbPU4A+4tojOtf+O0N0MAjDc8UFMiOZkzJz+F1mg3+Fdo7IhmFkrrhAZCRzUmQOv4ts+MfIfm0FVyEDua1AKN40pFfayD2eB6GkWGxdRxGZeKEF5FwsHA2dOI2i1yOKZ/yw4Kpvii9f+8PGdfwsmiTBjKKyEUZqIkG79KNKlro2FtZgcAYbmJm6hCsab1feHvE2XhNSB7nu3zhST0MEbHPLhvEovotytDweMV/es/dPvGamSg==

Good shout - thanks

**Comment by kamechan001** [comment]
> We were able to implement vertical steppers in v3 like this: https://play.vuetifyjs.com/#eNq9VMtOwzAQ/JWVL4BKEvo4oKqpQNy5wI1wcJytatVJLMdNkar+O+u4FBfaQ6Gqcoh3PZ4Zj6x9W7PGiORR67hdIhuzicVSK25xmlUAkzbiWnfLrhB1Zbms0GxbXbOxqDUaaKOyLlClGXOdjO0wISqaIy+C8z92JcmDlVYh0by6f8Zg3HK1dI0+FVhIy3OFkIT8yQGBg/IrWRX1CgppUFhZV8TaorFScEXkhJvXq+0N0pT0jhn1PN5v6G8KT5QRVnbPU4A+4tojOtf+O0N0MAjDc8UFMiOZkzJz+F1mg3+Fdo7IhmFkrrhAZCRzUmQOv4ts+MfIfm0FVyEDua1AKN40pFfayD2eB6GkWGxdRxGZeKEF5FwsHA2dOI2i1yOKZ/yw4Kpvii9f+8PGdfwsmiTBjKKyEUZqIkG79KNKlro2FtZgcAYbmJm6hCsab1feHvE2XhNSB7nu3zhST0MEbHPLhvEovotytDweMV/es/dPvGamSg==

Very nice, better than v-stepper-vertical
